src/collapse/collapse.js

```diff
@@ -11,28 +11,31 @@ angular.module('ui.bootstrap.collapse', [])
         }
 
         function expand() {
-          element.removeClass('collapse')
-            .addClass('collapsing')
-            .attr('aria-expanded', true)
-            .attr('aria-hidden', false);
+          $q.when(scope.$eval(attrs.expanding), function() {
```

Comment:
Might be better to use `$q.resolve`.

In addition, perhaps it is better to do
```js
$q.resolve(scope.$eval(attrs.expanding))
  .then(function() {
    ...
  });
```
since it is more in line with standard promise usage.